manual/cypher/cypher-docs/src/docs/dev/syntax/parameters.asciidoc

```diff
@@ -3,19 +3,32 @@
 
 Cypher supports querying with parameters.
 This means developers don't have to resort to string building to create a query.
-In addition to that, it also makes caching of execution plans much easier for Cypher.
+Additionally, parameters make caching of execution plans much easier for Cypher, thus leading to faster query execution times.
 
-Parameters can be used for literals and expressions in the `WHERE` clause, for the index value in the `START` clause, index queries, and finally for node/relationship ids.
-Parameters can not be used as for property names, relationship types and labels, since these patterns are part of the query structure that is compiled into a query plan.
+Parameters can be used for:
 
-Accepted names for parameters are letters and numbers, and any combination of these.
+* Literals and expressions (such as in the `WHERE` clause)
```

Comment:
I prefer the terse list version here since most of the entries are terse. The existence of a single long entry does not feel like sufficient reason to change all to long form.